datos/UserNames.js

```diff
@@ -4,4 +4,8 @@ module.exports = [
    { username : '' , password : '', reason : 'data was not received'},
    { username : '' , password : 'Demo123!', reason : 'username was not received'},
    { username : 'jsmith@demo.io' , password : '', reason : 'password was not received'}
-]
\ No newline at end of file
+]
+
+export const validUser = [
+   { username : 'jsmith@demo.io' , password : 'Demo123!' },
+];
```

Comment:
Great! :) 

By the way, I'd rename this file and _reason_ properties.

Reply:
Done